cmd/cli/metrics_enable.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/openservicemesh/osm/pkg/constants"
+)
+
+const metricsEnableDescription = `
+This command will enable metrics on all pods belonging to the given namespace
```

Comment:
Maybe: `This command will enable metrics for an existing Prometheus deployment for all pods...`

Reply:
It doesn't have to be existing, the user could deploy Prometheus whenever they want metrics collection to happen (post running this command)?

Reply:
Oh that's true. Maybe just add "this command does not deploy Prometheus"

Reply:
Added a note to the command documentation.